pyaedt/modeler/GeometryOperators.py

```diff
@@ -1448,3 +1462,354 @@ def orient_polygon(x, y, clockwise=True):
+        Iit the 2D version of the ``GeometryOperators.v_angle_sign`` considering vn = [0,0,1].
+        In case of opposite vectors, it returns an angle equal to 180deg (always positive).
+
+        Parameters
+        ----------
+        va : list
+            List of ``[x, y]`` coordinates for the first vector.
+        vb : list
+            List of ``[x, y]`` coordinates for the second vector.
+
+        righthanded : bool
+            If ``True`` the right-handed rotation from Va to Vb is considered.
+            If ``False`` the left-handed rotation from Va to Vb is considered. The default is ``True`.
+
+        Returns
+        -------
+        float
+            Angle in radians.
+
+        """
+        c = va[0] * vb[1] - va[1] * vb[0]
+
+        if righthanded:
+            return math.atan2(c, GeometryOperators.v_dot(va, vb))
+        else:
+            return math.atan2(-c, GeometryOperators.v_dot(va, vb))
+
+    @staticmethod
+    @aedt_exception_handler
+    def point_in_polygon(point, polygon):
+        """
+        Determine if a point is inside or outside a polyogn, both located on the same plane.
+        The method implements the radial algorithm (https://es.wikipedia.org/wiki/Algoritmo_radial)
+
+        point : list
+            List of ``[x, y]`` coordinates.
+        polygon : list
+            [[x1, x2, ..., xn],[y1, y2, ..., yn]]
+
+        Returns
+        -------
+        int
+            -1 When the point is outside the polygon.
+            0 When the point is exactly on one of the sides of the polygon.
+            1 When the point is inside the polygon.
+        """
+        # fmt: off
+        tol = 1e-8
+        assert len(point) == 2, "point must be a list in the form [x, y]"  # pragma: no cover
+        pl = len(polygon[0])
+        assert len(polygon[1]) == pl, "Polygon x and y lists must be the same length"  # pragma: no cover
+        asum = 0
+        for i in range(pl):
+            vj = [polygon[0][i-1], polygon[1][i-1]]
+            vi = [polygon[0][i], polygon[1][i]]
+            if GeometryOperators.points_distance(point, vi) < tol:
+                return 0  # point is one of polyline vertices
+            vpj = GeometryOperators.v_points(point, vj)
+            vpi = GeometryOperators.v_points(point, vi)
+            a = GeometryOperators.v_angle_sign_2D(vpj, vpi)
+            if abs(abs(a) - math.pi) < tol:
+                return 0
+            asum += a
+        if abs(asum) < tol:
+            return -1
+        elif abs(asum - 2*math.pi) < tol:
+            return 1
+        else:  # pragma: no cover
+            raise Exception("Unexpected error!")
+        # fmt: on
+
+    @staticmethod
+    @aedt_exception_handler
+    def is_point_in_polygon(point, polygon):
+        """
+        Determine if a point is inside or outside a polyogn, both located on the same plane.
+        The method implements the radial algorithm (https://es.wikipedia.org/wiki/Algoritmo_radial)
```

Comment:
Should we be using the english wiki since docs are in english?

Reply:
There is no English version of this page. There is only one paper about this algorithm and it is in Spanish too. I had to use google translator myself.